manager/gui/src/lib/components/SgtAdvanceSearch/SgtAdvanceSearch.vue

```diff
@@ -76,6 +76,7 @@
                     v-model="row.value"
                     outlined
                     dense
+                    :disabled="row.required"
```

Comment:
disabled is not directly proportional with required

Reply:
Added a new property 'editable' to handle disabling inputs.